TheWarWithin/MageFire.lua

```diff
@@ -865,10 +896,16 @@ spec:RegisterAuras( {
 
 spec:RegisterGear( "tier29", 200318, 200320, 200315, 200317, 200319 )
 
+local TriggerHyperthermia = setfenv( function()
+    applyBuff( "Hyperthermia", 2 + ( buff.lingering_embers.stacks * 1 ) )
```

Comment:
Hyperthermia should be lowercase.

Reply:
...and `* 1` is not needed.

Reply:
previously it was 0.5 before this patch, I think that's why I updated it to 1 lol